packages/google_maps_flutter/lib/src/marker.dart

```diff
@@ -83,9 +129,24 @@ class MarkerOptions {
     this.rotation,
     this.visible,
     this.zIndex,
-  });
+  }) : assert(alpha == null || 0.0 <= alpha && alpha <= 1.0);
```

Comment:
Add grouping parenthesis for clarity: `assert(alpha == null || (0.0 <= alpha && alpha <= 1.0));`